subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/SyncTaskIntegrationTest.groovy

```diff
@@ -394,36 +393,39 @@ class SyncTaskIntegrationTest extends AbstractIntegrationSpec {
         ins.close()
     }
 
-    @ToBeImplemented
     @Issue("https://github.com/gradle/gradle/issues/9586")
-    @Requires(TestPrecondition.CASE_INSENSITIVE_FS)
     def "change in case of input file still syncs properly"() {
```

Comment:
Should I add a test that changed the folder's case, too?

Reply:
I thank that would be good.

Reply:
Done